gcp_variant_transforms/pipeline_common.py

```diff
@@ -274,3 +314,15 @@ def generate_unique_name(job_name):
   return '-'.join([job_name,
                    datetime.now().strftime('%Y%m%d-%H%M%S'),
                    str(uuid.uuid4())])
+
+
+class FusionBreak(beam.PTransform):
```

Comment:
Do you mean `ReadFromVcf` and `ReadAllFromVcf`? I didn't do it because I don't want to add the same code into both of them. Does this make sense to you?

Reply:
As discussed offline, moved `FusionBreak` into a separate script.